Ladybird/Android/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+          xmlns:tools="http://schemas.android.com/tools"
+          android:installLocation="auto"
+          android:versionCode="001"
+          android:versionName="head">
+<supports-screens android:anyDensity="true"
+                  android:largeScreens="true"
+                  android:normalScreens="true"
+                  android:smallScreens="true"/>
+<application
+        android:dataExtractionRules="@xml/data_extraction_rules"
+        android:fullBackupContent="@xml/backup_rules"
+        android:icon="@mipmap/ic_launcher"
+        android:roundIcon="@mipmap/ic_launcher_round"
+        android:supportsRtl="true"
+        android:theme="@style/Theme.Ladybird"
+        tools:targetApi="31"
+        android:hardwareAccelerated="true"
+        android:label="ladybird"
+        android:requestLegacyExternalStorage="true"
+        android:allowNativeHeapPointerTagging="false"
+        android:allowBackup="true"
+        android:fullBackupOnly="false"
+        android:enableOnBackInvokedCallback="true">
+    <activity
+            android:name=".LadybirdActivity"
+            android:configChanges="orientation|uiMode|screenLayout|screenSize|smallestScreenSize|layoutDirection|locale|fontScale|keyboard|keyboardHidden|navigation|mcc|mnc|density"
+            android:label="ladybird"
+            android:launchMode="singleTop"
+            android:screenOrientation="unspecified"
+            android:exported="true">
+        <intent-filter>
+            <action android:name="android.intent.action.MAIN" />
+            <category android:name="android.intent.category.LAUNCHER" />
```

Comment:
Shouldn't it also accept `android.intent.action.VIEW` for actually handling URLs?

Reply:
I think this is missing a lot of intents, but we'll find them as features are added. Same with permissions (missing NETWORK at least)